src/libraries/System.Text.Json/tests/TrimmingTests/Helper.cs

```diff
@@ -90,16 +90,18 @@ public static bool AssertCollectionAndSerialize<T>(object obj, string json)
         }
     }
 
-    internal class MyClass
+    public class MyClass
```

Comment:
Is this change necessary?

Reply:
Yes - it helps avoid `warning CS0649: Field is never assigned to, and will always have its default value`